File: 3rd_party/adios/testing/install/C/main_nompi.c
/*
 * Distributed under the OSI-approved Apache License, Version 2.0.  See
 * accompanying file Copyright.txt for details.
 */

#include <adios2_c.h>

#if ADIOS2_USE_MPI
#error "ADIOS2_USE_MPI is true for source not using ADIOS2 MPI bindings"
#endif

#include <stdio.h>

int main(void)
{
    adios2_adios *adios = adios2_init_serial();
    if (!adios)
    {
        fprintf(stderr, "adios2_init_serial() failed\n");
        return 1;
    }
    adios2_finalize(adios);
    return 0;
}

File: 3rd_party/adios/testing/install/Fortran/main_nompi.f90
program adios_fortran_test
  use adios2
  integer :: ierr, irank, isize
  type(adios2_adios) :: adios

  call adios2_init(adios, ierr)
  call adios2_finalize(adios, ierr)
end program adios_fortran_test
